[2022-08-24T19:41:35.912+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: another_dag.accurate manual__2022-08-24T19:41:33.001653+00:00 [queued]>
[2022-08-24T19:41:35.919+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: another_dag.accurate manual__2022-08-24T19:41:33.001653+00:00 [queued]>
[2022-08-24T19:41:35.921+0000] {taskinstance.py:1368} INFO - 
--------------------------------------------------------------------------------
[2022-08-24T19:41:35.922+0000] {taskinstance.py:1369} INFO - Starting attempt 1 of 1
[2022-08-24T19:41:35.923+0000] {taskinstance.py:1370} INFO - 
--------------------------------------------------------------------------------
[2022-08-24T19:41:35.963+0000] {taskinstance.py:1389} INFO - Executing <Task(BashOperator): accurate> on 2022-08-24 19:41:33.001653+00:00
[2022-08-24T19:41:35.971+0000] {standard_task_runner.py:52} INFO - Started process 24034 to run task
[2022-08-24T19:41:35.974+0000] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'another_dag', 'accurate', 'manual__2022-08-24T19:41:33.001653+00:00', '--job-id', '29', '--raw', '--subdir', 'DAGS_FOLDER/another_dag.py', '--cfg-path', '/tmp/tmpf6n0ntpi', '--error-file', '/tmp/tmpj7eviovf']
[2022-08-24T19:41:35.976+0000] {standard_task_runner.py:80} INFO - Job 29: Subtask accurate
[2022-08-24T19:41:36.023+0000] {task_command.py:371} INFO - Running <TaskInstance: another_dag.accurate manual__2022-08-24T19:41:33.001653+00:00 [running]> on host 9ab687ae92a7
[2022-08-24T19:41:36.075+0000] {taskinstance.py:1583} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=another_dag
AIRFLOW_CTX_TASK_ID=accurate
AIRFLOW_CTX_EXECUTION_DATE=2022-08-24T19:41:33.001653+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-08-24T19:41:33.001653+00:00
[2022-08-24T19:41:36.077+0000] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-24T19:41:36.080+0000] {subprocess.py:74} INFO - Running command: ['/bin/bash', '-c', "echo 'accurate'"]
[2022-08-24T19:41:36.093+0000] {subprocess.py:85} INFO - Output:
[2022-08-24T19:41:36.099+0000] {subprocess.py:92} INFO - accurate
[2022-08-24T19:41:36.102+0000] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-24T19:41:36.125+0000] {taskinstance.py:1412} INFO - Marking task as SUCCESS. dag_id=another_dag, task_id=accurate, execution_date=20220824T194133, start_date=20220824T194135, end_date=20220824T194136
[2022-08-24T19:41:36.147+0000] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-24T19:41:36.170+0000] {local_task_job.py:279} INFO - 0 downstream tasks scheduled from follow-on schedule check
